etp-backend/src/main/clj/solita/etp/service/valvonta_kaytto/toimenpide.clj

```diff
@@ -23,8 +23,12 @@
    11 :decision-order-notice-bailiff
    12 :decision-order-waiting-for-deadline
    14 :penalty-decision-hearing-letter
+   15 :change-when-implement-penalty-decision-actual-decision
```

Comment:
Täytyi lisätä kaikille kannassa oleville toimenpiteille jokin placeholder tähän. Näitä on nyt siis idt 15, 18, 21. Laitoin eteen **change-when-implement**.

Id 19 on tämän ticketin toimenpide.